packages/cli/package.json

```diff
@@ -54,7 +54,7 @@
     "workflow"
   ],
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=6.9"
```

Comment:
why 6.9 here?

Reply:
i pushed the fix. but it still hasn't shown up 🤔